web/styles/lightbox.css

```diff
@@ -50,19 +50,23 @@
     }
 
     .exit {
-        flex-shrink: 0;
-
         color: hsl(0deg 0% 100% / 80%);
-        font-size: 2rem;
-        line-height: 1.8rem;
-
-        transform: scaleY(0.75);
-        font-weight: 300;
 
         opacity: 0;
         pointer-events: none;
         cursor: pointer;
         transition: opacity 0.2s ease;
+
+        position: relative;
```

Comment:
I would much prefer a flexbox rewrite in this area and not use positioning like this. 

I also don't understand the use of `rem` units on the height and width here. Shouldn't this scale with everything else?